src/app/auth/account/account.component.html

```diff
@@ -125,10 +127,10 @@ <h2 class="text-start pt-3 px-3 h6 organization">Organizations</h2>
             </ng-container>
             <ng-container *ngIf="isAccountViewTypeDatasets">
                 <app-datasets-tab
-                    [datasets]="datasets"
+                    [datasets]="datasets.datasets"
```

Comment:
I suggest to rename datasets, for example to `data`, to avoid name duplication

Reply:
Ok, renamed